### ISSUE TITLE
Mount dropbox files

### ISSUE THREAD

**Issue by kousuke-nakano** [ish-app/ish#1838]
Dear developers and users,

I am trying to mount dropbox files on iPad.

- iPad Pro (10.5-inch)
- iOS: v15.5
- iSH.app v1.2.3 (build 298), Alpine v3.14
- Dropbox.app v286.2.2 

Although I successfully mounted external directories such as "iCloud Drive" and "on My iPad" (#230),
it did not work for dropbox files. I used the following command.

`# mount -t ios . /mnt`

The problem is that I could not select the dropbox icon on the iOS file browser launched by the above command because the icon was grey (instead, the icons of "iCloud Drive" and "on My iPad" were black). The contents of my dropbox files can be accessed from File.app. Therefore, I do not think this is a permission problem.

I was wondering if someone would know a solution. I would appreciate any comment.

Best regards.

Kosuke Nakano

**Comment by philipsd6** [comment]
For me, when I do the mount command, I can browse my Dropbox files, but the "Done" link/button is disabled (gray.)

**Comment by kousuke-nakano** [comment]
> but the "Done" link/button is disabled (gray.)

Thank you for your reply. yes, this is the same. The link is disable(grey).

**Comment by ZGGSONG** [comment]
Yes, I'm trying to mount my Synology Drive which is also greyed out not sure how to go about fixing this

**Comment by upintheairsheep** [comment]
Try rclone
